client/src/components/layout/Navigation.js

```diff
@@ -101,26 +99,5 @@ class Navigation extends Component {
     }
 }
 
-const query = {
-    url: "/api/0/users/self",
-    transform: (json, text) => {
-        return { user: json }
-    },
-    update: {
-        user: (prevUser, user) => user
-    }
-}
-
-Navigation = compose(
-    connect(state => {
-        return {
-            user: {
-                data: state.entities.user,
-                metadata: { isFinished: querySelectors.isFinished(state.queries, query) },
-            }
-        }
-    }),
-    connectRequest(props => ([query]))
-)(Navigation)
-
-export default Navigation
+const query = gql`query { isAuthenticated }`
```

Comment:
This is an example of retrieving data from the backed via a graphql query. Here we're trying to see if the user is logged in or not.